scio-core/src/main/scala/com/spotify/scio/sql/Query.scala

```diff
@@ -207,6 +114,178 @@ object Query {
 
 }
 
+trait Query2[A, B, C] extends ((SCollection[A], SCollection[B]) => SCollection[C]) {
+  def query: String
+}
+
+object Query2 {
```

Comment:
Does this mean that we can only join 2 `SCollections` ?

Reply:
yeah, currently beam only allow's join of 2 collections

Bounded input JOIN bounded input
Unbounded input JOIN unbounded input
Unbounded input JOIN bounded input